### ISSUE TITLE
Need conditional logic for older versions of notation

### ISSUE THREAD

**Issue by dtzar** [Duffney/setup-notation#3]
When you use notation 0.12.0-beta.1 - it fails ([see pipeline run for full context ](https://github.com/dtzar/net-monitor/actions/runs/3520357621/jobs/5901198777) if desired) because the syntax has changed.

```
Error: unknown flag: --name
node:child_process:903
    throw err;
    ^

Error: Command failed: notation key add --name wabbit-networks-io --id *** --default --plugin azure-kv
Error: unknown flag: --name

    at checkExecSyncError (node:child_process:826:11)
    at execSync (node:child_process:900:15)
    at addPluginCert (/home/runner/work/_actions/Duffney/setup-notation/v0.1.1/dist/index.js:163:24)
    at /home/runner/work/_actions/Duffney/setup-notation/v0.1.1/dist/index.js:85:9
    at /home/runner/work/_actions/Duffney/setup-notation/v0.1.1/dist/index.js:7161:26
    at FSReqCallback.oncomplete (node:fs:188:23) {
  status: 1,
  signal: null,
  output: [ null, '', 'Error: unknown flag: --name\n' ],
  pid: 2059,
  stdout: '',
  stderr: 'Error: unknown flag: --name\n'
}
```

```
 notation key add -h
Add key to signing key list

Usage:
  notation key add --plugin <plugin_name> [flags] <key_name>

Flags:
  -d, --default                     mark as default
  -h, --help                        help for add
      --id string                   key id (required if --plugin is set)
  -p, --plugin string               signing plugin name
  -c, --plugin-config stringArray   {key}={value} pairs that are passed as it is to a plugin, refer plugin's documentation to set appropriate values
```

**Comment by duffney** [comment]
I'll get started on adding in that logic to the action. Thank you for opening the issue. :)

**Comment by duffney** [comment]
Hi @dtzar, I've updated the logic of the action to account for the changes introduced in version 0.12.0-beta.1 with the parameters changing. Instead of addressing the changing parameters within a single continuous release, I've add tags that support specific versions of the NotationCli. For example, tag 0.11.0 pairs with the minor version of 0.11.X of the NotationCLi and likewise for version 0.12.0 of the setup_notation GitHub action. Please let me know if you have any questions.

TLDR: Duffney/setup_notation@0.11.0 supports 0.11.X of the Notation Cli & @0.12.0 supports 0.12.0-beta.1 and higher asumming no more param changes occur within that minor version rev.
